services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
Consider adding exception handling for `MovieSession.DoesNotExist` to handle cases where a movie session with the given `movie_session_id` does not exist. This will prevent the application from crashing and allow you to provide a more user-friendly error message.